android-client-sdk/consumer-rules.pro

```diff
@@ -0,0 +1,9 @@
+# Add build specific ProGuard rules here.
+# These rules will be pulled when a consumer installs the DevCycle Android SDK.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+-keepclassmembers enum com.devcycle.sdk.** { *; }
```

Comment:
why this

Reply:
good point, going to narrow it down to just keep the models instead of the entire SDK